### ISSUE TITLE
Math error in DNG lens correction?

### ISSUE THREAD

**Issue by agriggio** [artpixls/ART#177]
**[Original report](https://bitbucket.org/agriggio/art/issues/177) by Paolo DePetrillo (Bitbucket: [paolodepetrillo1](https://bitbucket.org/paolodepetrillo1), ).**

----------------------------------------

Comparing to the DNG spec, I think there’s a \* in this line that should be a \+, in ExifLensCorrection::correctDistortion:

```
    const float f = dist_[0] + r2 * (dist_[1] + r2 * (dist_[2] * r2 * dist_[3]));
```

should be:

```
    const float f = dist_[0] + r2 * (dist_[1] + r2 * (dist_[2] + r2 * dist_[3]));
```

‌

Haven’t actually tested this myself as I just realized my phone camera outputs a no-op lens correction and doesn’t need it..

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

looks like a typo indeed, thanks for reporting!

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

fix typo in DNG distortion correction

Fixes #177
